.github/workflows/scripts/issues/titleCleanup.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+const regexes = {
+  // covers 'dataGrid' in all variants, as well as with suffixes (-pro/-premium/etc)
+  dataGrid: /\s?\[\s?(data\s?grid[-\sA-Za-z]*)]/gi,
```

Comment:
```suggestion
  dataGrid: /\s?\[\s?(data[\s-]?grid[-\sA-Za-z]*)]/gi,
```

I see datagrid spelled as `data-grid` sometimes as well